### ISSUE TITLE
but-em-buttons

### ISSUE THREAD

**Issue by alexhealy** [alexhealy/but-em-buttons#1]
Hi Thomas

Markbot doesnt seem to be happy. I've made the photos the correct sizes and compressed them within the imageoptim app however markbot keeps telling me the opposite and won't pass, it keeps loading and won't seem to finalize the issue. 
@thomasjbradley

Alex

**Comment by thomasjbradley** [comment]
Try restarting your computer to see if that helps Markbot finish.

Did you resave the stars image from Photoshop, using Save for Web before running it through ImageOptim?

**Comment by alexhealy** [comment]
Yes I've restarted my computer a few times. I re-downloaded it twice and repeated the same steps to see if i missed anything but seems to still not work. And yes I resaved the stars image from photoshop using the save for web and then placed it in the imageoptim app.

@thomasjbradley

________________________________

**Comment by thomasjbradley** [comment]
When you saved from Photoshop did you choose PNG/24 or PNG/8 because there are only a few colours in the graphic so PNG/8 should work well.
